### PR TITLE
PR: modification of create article page to fix small bug

### PR DESCRIPTION
**Pull request by creative2113**
please verify this pr and give me a good review.
thank u

### CODE REVIEW

File: client/src/components/Input/SelectBoxSmall.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ function SelectBoxSmall(props) {
         value={props.value}
         onChange={props.onChange}
         name={props.name}
+        required
       >
+        <option></option>
```
Comment (shinevue) [inline client/src/components/Input/SelectBoxSmall.js:14]: Why need empty option tag?

File: client/src/pages/Article/NewArticle.jsx [hunk 2]
```diff
@@ -27,27 +31,39 @@ const NewArticle = () => {
   });
 
   useEffect(() => {
-    if (selected_id.id !== 0) {
+    if (selected_id.id !== 0 && selected_id.id !== "0") {
```
Comment (shinevue) [inline client/src/pages/Article/NewArticle.jsx:34]: When id will be same as "0" of string?

**Review by shinevue — CHANGES_REQUESTED** [review]
Good job
But let's solve this issue

**Review by creative2113 — COMMENTED** [review]
good style code.

File: client/src/components/Input/SelectBoxSmall.js (hunk 1 above)
Comment (BlackGhost2693) [reply client/src/components/Input/SelectBoxSmall.js:14]: I think with you

**Review by BlackGhost2693 — CHANGES_REQUESTED** [review]
Hi, @creative2113 
Good job
Awesome

**Review by creative2113 — COMMENTED** [review]
don't mind

**Comment by creative2113** [comment]
> Hi, @creative2113 Good job Awesome

thank you.

File: client/src/components/Input/SelectBoxSmall.js (hunk 1 above)
Comment (creative2113) [reply client/src/components/Input/SelectBoxSmall.js:14]: good idea.

**Review by creative2113 — COMMENTED** [review]
go ahead

File: README.md [hunk 3]
```diff
@@ -59,5 +59,5 @@
 #### [Luckystar](https://github.com/techietrend)
 #### [Achilles](https://github.com/oleh1010)
 #### [Chivalrousdev](https://github.com/chivalrousdev)
-#### [ScottKing](https://github.com/creative2113)
```
Comment (joellake992) [inline README.md:62]: Rename your nickname

**Review by joellake992 — CHANGES_REQUESTED** [review]
Change your nickname with Paskal case and no-number

File: README.md (hunk 3 above)
Comment (creative2113) [reply README.md:62]: yes, I see.

Comment (creative2113) [reply README.md:62]: I have changed my nick name.
please check this PR.

**Review by creative2113 — COMMENTED** [review]
good

**Review by shinevue — APPROVED** [review]
Completed

**Review by joellake992 — APPROVED** [review]
Excellent.

**Review by BlackGhost2693 — APPROVED** [review]
thank you

**Pull request merged**